server/errors.py

```diff
@@ -0,0 +1,16 @@
+class HTTPError(Exception):
+    def __init__(self, message=""):
+        super()
+        self.message = message
```

Comment:
I think this is supposed to be a `pass` statement? The merge might've worked out weirdly.

Reply:
I havent merged in changes from #54 recently - this branch is only up-to-date with half the commits from that PR. I believe it hasn't been updated with the Error-related changes and new handling. I will merge these changes soon.